draft-ietf-rats-corim.md

```diff
@@ -1439,17 +1439,17 @@ If the merged measurement-value-map contains duplicate codepoints and the
 measurement values are not equivalent then the verifier SHALL report
 an error and stop validation processing.
 
-### Accepted Claims Set Initialisation
+### Accepted Claims Set Initialization
 
-The Accepted Claims Set is initialised with cryptographically verified Evidence
-from the Attestation Environments.
+The Accepted Claims Set is initialized by copying Evidence claims from the authenticated Attester's Target Environments
+into the Verifier's Accepted Claims Set. By default, all Attester claims are allowed, unless disallowed by Reference Claims
+that do not satisfy appraisal checks.
```

Comment:
```suggestion
into the Verifier's Accepted Claims Set.
```

Reply:
+1
Initialization Step should not include "disallow" step content